src/hooks/useLocale/useLocale.ts

```diff
@@ -0,0 +1,18 @@
+import { useContext } from 'react';
+import { IntlShape, useIntl } from 'react-intl';
+import { LocaleContext } from 'context/locale/localeContext/LocaleContext';
```

Comment:
According to the eslint rules, there should be an empty line between external libs imports and the local ones, right? Are you sure, eslint is configured properly?

Reply:
I am. When I changed `useIntl` to `useLocale`, it marked me code above as error.